CHANGELOG.rst

```diff
@@ -1,6 +1,44 @@
 Changelog
 =========
 
+1.10.0 (2022-09-29)
+-------------------
+
+New Operators
+"""""""""""""
+
+This release adds the following 2 new async sensors/operators:
+
+.. list-table::
+   :header-rows: 1
+
+   * - Operator/Sensor Class
+     - Import Path
+     - Example DAG
+
+   * - ``ExternalDeploymentTaskSensorAsync``
+     - .. code-block:: python
+
+        from astronomer.providers.http.sensors.external_deployment_task import ExternalDeploymentTaskSensorAsync
+     - `Example DAG <https://github.com/astronomer/astronomer-providers/blob/main/astronomer/providers/http/example_dags/example_external_deployment_task_sensor.py>`__
+
+   * - ``SFTPSensorAsync``
+     - .. code-block:: python
+
+        from astronomer.providers.sftp.sensors.sftp import SFTPSensorAsync
+     - `Example DAG <https://github.com/astronomer/astronomer-providers/blob/main/astronomer/providers/sftp/example_dags/example_sftp.py>`__
+
+Bug Fixes
+"""""""""
+
+- Use `poke_interval` instead of `poll_interval` for all async sensor
```

Comment:
This was deprecation not bug I feel we should create one more section for deprecation WDYT?